cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,74 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import SettinsPageObject from '../support/pages/settingPageObjects';
+import homePageObject from '../support/pages/home.pageObject';
 
-  });
+const homePage = new homePageObject();
+const signInPage = new SignInPageObject();
+const settingsPage = new SettinsPageObject();
 
-  beforeEach(() => {
+const userData = {
+  username: 'riot',
```

Comment:
Consider using faker.js to generate fake data for tests. It adds more variability to the tests and ensures that the code can handle different inputs.